### PR TITLE
Update GameVersionHelper.cs Bunch more Title IDs and seperated Retail from Debug, Beta and Demo builds

### PR DESCRIPTION
**Pull request by FridiNaTor1**
Added a whole bunch more of Title IDs removed duplicates, fixed formatting, and grouped Retail by itself and Debug, Beta and Demo under a comment. Also grouped PS4 and PS3 versions of LBP 3 by itself

### CODE REVIEW

**Comment by FridiNaTor1** [comment]
There was also a LBP PSP Title ID that was listed under LBP 1 and not LBP PSP, Which I have moved

File: ProjectLighthouse/Helpers/GameVersionHelper.cs [hunk 1]
```diff
@@ -45,54 +52,105 @@ public class GameVersionHelper
...
+        "NPHA80113",
+        "NPHA80125",
+        "NPJA90152",
+        "NPUA70127",
+        "NPUA70169",
+        "NPUA70174",
+	//HUB
+        "BCET70055",
+        "NPEA00449",
+        "NPHA80261",
```
[the remark this answers is not in the document]
Comment (FridiNaTor1) [reply ProjectLighthouse/Helpers/GameVersionHelper.cs:76]: Yes

File: ProjectLighthouse/Helpers/GameVersionHelper.cs [hunk 2]
```diff
@@ -17,17 +17,24 @@ public class GameVersionHelper
...
-        "NPHG00035",
         "NPEA00241",
         "NPUA98208",
+        "NPHA80092",
+        "BCKS10059",
+        "BCKS10088",
+        "BCUS70030",
+        "NPJA00052",
+        "NPUA80472",
+	//Debug, Beta and Demo
```
[the remark this answers is not in the document]
Comment (FridiNaTor1) [reply ProjectLighthouse/Helpers/GameVersionHelper.cs:31]: Indent? As Identify?

[the remark this answers is not in the document]
Comment (jvyden) [reply ProjectLighthouse/Helpers/GameVersionHelper.cs:31]:
```suggestion
	    //Debug, Beta and Demo
```

[the remark this answers is not in the document]
Comment (FridiNaTor1) [reply ProjectLighthouse/Helpers/GameVersionHelper.cs]: Gonna install Visual Studio Code, I was using Notepad++ and for some reason it looks different on it, compared to GitHub

[the remark this answers is not in the document]
Comment (FridiNaTor1) [reply ProjectLighthouse/Helpers/GameVersionHelper.cs]: Should be fixed now :)

**Review by jvyden — APPROVED** [review]

**Pull request merged**